libraries/Microsoft.Bot.Schema/Activity.cs

```diff
@@ -131,7 +131,12 @@ public Activity()
+        /// <param name="callerId">A string containing an IRI identifying the 
+        /// caller of a bot. This field is not intended to be transmitted
+        /// over the wire, but is instead populated by bots and clients based on 
+        /// cryptographically verifiable data that asserts the identity of the 
+        /// callers (e.g. tokens).</param>
+        public Activity(string type = default(string), string id = default(string), System.DateTimeOffset? timestamp = default(System.DateTimeOffset?), System.DateTimeOffset? localTimestamp = default(System.DateTimeOffset?), string serviceUrl = default(string), string channelId = default(string), ChannelAccount from = default(ChannelAccount), ConversationAccount conversation = default(ConversationAccount), ChannelAccount recipient = default(ChannelAccount), string textFormat = default(string), string attachmentLayout = default(string), IList<ChannelAccount> membersAdded = default(IList<ChannelAccount>), IList<ChannelAccount> membersRemoved = default(IList<ChannelAccount>), IList<MessageReaction> reactionsAdded = default(IList<MessageReaction>), IList<MessageReaction> reactionsRemoved = default(IList<MessageReaction>), string topicName = default(string), bool? historyDisclosed = default(bool?), string locale = default(string), string text = default(string), string speak = default(string), string inputHint = default(string), string summary = default(string), SuggestedActions suggestedActions = default(SuggestedActions), IList<Attachment> attachments = default(IList<Attachment>), IList<Entity> entities = default(IList<Entity>), object channelData = default(object), string action = default(string), string replyToId = default(string), string label = default(string), string valueType = default(string), object value = default(object), string name = default(string), ConversationReference relatesTo = default(ConversationReference), string code = default(string), System.DateTimeOffset? expiration = default(System.DateTimeOffset?), string importance = default(string), string deliveryMode = default(string), IList<string> listenFor = default(IList<string>), IList<TextHighlight> textHighlights = default(IList<TextHighlight>), SemanticAction semanticAction = default(SemanticAction), string localTimezone = default(string), string callerId = default(string))
```

Comment:
We can't do this. Adding a constructor parameter is a breaking change. 

What we do about this, I'm not sure. :(

Reply:
Have to make this a new ctor with the additional parameter required, then have 1 call the other.

Reply:
Removed the callerId parameter. 

This constructor is unwieldy.